cpp/inc/bond/ext/grpc/detail/service_call_data.h

```diff
@@ -69,7 +69,7 @@ struct service_unary_call_data : io_manager_tag
           _cq(cq),
           _threadPool(threadPool),
           _cb(cb),
-          _receivedCall(std::make_shared<unary_call_impl<TRequest, TResponse>>())
+          _receivedCall(new unary_call_impl<TRequest, TResponse>)
```

Comment:
`make_unique`?

Reply:
`std::make_unique` was only added in C++14.